apps/lfx-one/src/app/modules/committees/committee-manage/committee-manage.component.ts

```diff
@@ -379,6 +379,8 @@ export class CommitteeManageComponent {
       sso_group_name: committee.sso_group_name,
       website: committee.website,
       joinable: false,
+      member_visibility: committee.member_visibility || 'hidden',
```

Comment:
Why not convert this to a boolean and rename the field? Are there multiple values that this field is expected to have? If so, can we set this to a specific type with a limited set of values? Looking further down, this is tightly related to Angular's type system and likely a convention that's traditionally used.